app/serializers/user_timezone_serializer.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
 class UserTimezoneSerializer < BasicUserSerializer
-  attributes :timezone
+  attributes :timezone, :on_holiday
+
+  def on_holiday
```

Comment:
The name of this class is `UserTimezoneSerializer`, so `on_holiday` field doesn't fit in here, it has nothing to do with timezones. We either need to rename the class or we need to find another place for it.